### PR TITLE
Fix tmpseek when execution times is in the command

### PR DESCRIPTION
**Pull request by mytbk**
Before applying this patch, running `times cmd @ tmpoff` will seek to
*tmpoff* and run *cmd* the first time, restore to the original offset,
then run *cmd* at the original offset for the rest of the time. This
will break the `wxs` command which changes the offset after each run.

In this patch, when tmpseek is used in `[rep] cmd @ tmpoff`:
1. save core->offset, seek to tmpoff
2. run the command rep times, do not restore the offset after a run
3. then restore core->offset even if cmd changes offset

Test:
- s addr @ tmpoff: this should not change the offset
- rep wxs bytes @ tmpoff: this should write bytes *rep* times
  starting at *tmpoff*, then the offset is restored

More tests are needed

### CODE REVIEW

**Comment by radare** [comment]
Good catch! Can you write a test case for the regressions testsuite?

Thanks!

File: libr/core/cmd.c [hunk 1]
```diff
@@ -1608,6 +1612,13 @@ static int r_core_cmd_subst(RCore *core, char *cmd) {
 		char *cr = strdup (cmdrep);
 		core->break_loop = false;
 		ret = r_core_cmd_subst_i (core, cmd, colon);
+		/*
+		 * r_core_cmd_subst_i will set core->tmpseek when run cmd the first time
+		 * if `@ tmpoff` is used, keep core->tmpseek in the remaining runs
+		 */
+		if (core->tmpseek)
```
Comment (radare) [inline libr/core/cmd.c]: Add {} braces

File: libr/core/cmd.c [hunk 2]
```diff
@@ -1626,6 +1637,9 @@ static int r_core_cmd_subst(RCore *core, char *cmd) {
 		}
 		free (cr);
 	}
+	if (tmpseek)
```
Comment (radare) [inline libr/core/cmd.c]: Add braces

**Comment by mytbk** [comment]
changes updated, tests are added to: https://github.com/radare/radare2-regressions/pull/1078

**Pull request merged**